src/exampleTypes/TagsQuerySearchBar.js

```diff
@@ -29,6 +29,7 @@ let buttonStyle = {
 
 let AnimatedButton = ({ disabled, style, className, ...props }) => (
   <Button
+    data-testid="button-main-search"
```

Comment:
Instead why not just add the same `data-path` to `SearchBar` and select the input and the search button from there?

Reply:
Done.